test/Microsoft.Azure.WebJobs.Host.EndToEndTests/EventHubEndToEndTests.cs

```diff
@@ -148,10 +151,22 @@ public static void SendEvents_TestHub2(int numEvents, string input, [EventHub(Te
                 }
             }
 
-            public static void ProcessMultipleEvents([EventHubTrigger(TestHub2Name)] string[] events)
+            public static void ProcessMultipleEvents([EventHubTrigger(TestHub2Name)] string[] events,
+                string[] partitionKeyArray, DateTime[] enqueuedTimeUtcArray, IDictionary<string, object>[] propertiesArray,
```

Comment:
Here is the new binding support in action. In Azure Functions for a Node.js function, this binding data will be accessible via:

```javascript
module.exports = function (context, events) {
    for (int i = 0; i < events.length; i++
    {
        var properties = context.bindingData.propertiesArray[i];
        . . .
    }

    . . .
}
```